lib/MusicBrainz/Script/MBDump.pm

```diff
@@ -157,7 +163,7 @@ sub make_tar {
     my $compress_command;
     if ($compression) {
         $compress_command = "$compression";
-        $compress_command .= ' --threads=0' if $compression eq 'xz';
+        $compress_command .= ' --threads=$compression_threads' if $compression eq 'xz';
```

Comment:
A declaration for `my $compression_threads` seems to be missing here.

Reply:
Thanks, fixed that and changed to double quotes too.